hadoop-hdds/container-service/src/test/java/org/apache/hadoop/ozone/container/keyvalue/TestKeyValueContainerMetadataInspector.java

```diff
@@ -172,7 +269,8 @@ public void inspectThenRepairOnCorrectContainer(
    */
   public void inspectThenRepairOnIncorrectContainer(
       KeyValueContainerData containerData, int createdBlocks, int setBlocks,
-      int setBytes) throws Exception {
+      int setBytes, int deleteCount, long numDeletedLocalIds,
+      boolean shouldRepair) throws Exception {
```

Comment:
`shouldRepair` is for the case `deleteTransactionCount == 0`.  If we change to repair for any `deleteTransactionCount`, then we can remove `shouldRepair`.

Reply:
> New parameters should be added to javadoc

Come on, javadoc for unit test?  Are you serious?

Reply:
I was just suggesting we keep the existing doc and parameter list consistent since we were already making changes here, but if you are against it it's not a blocking issue.